src/pages/ServiceList/ServiceListPage.tsx

```diff
@@ -1,43 +1,20 @@
 import * as React from 'react';
-import { Alert } from 'patternfly-react';
 import ServiceListComponent from './ServiceListComponent';
+import * as MessageCenter from '../../utils/MessageCenter';
 
-type ServiceListState = {
-  alertVisible: boolean;
-  alertDetails: string;
-};
+type ServiceListState = {};
 
 type ServiceListProps = {
   // none yet
 };
 
 class ServiceListPage extends React.Component<ServiceListProps, ServiceListState> {
-  constructor(props: ServiceListProps) {
-    super(props);
-
-    this.state = {
-      alertVisible: false,
-      alertDetails: ''
-    };
-  }
-
   handleError = (error: string) => {
-    this.setState({ alertVisible: true, alertDetails: error });
-  };
-
-  dismissAlert = () => {
-    this.setState({ alertVisible: false });
+    MessageCenter.add(error);
   };
 
   render() {
     let alertsDiv = <div />;
-    if (this.state.alertVisible) {
-      alertsDiv = (
-        <div>
-          <Alert onDismiss={this.dismissAlert}>{this.state.alertDetails.toString()}</Alert>
-        </div>
-      );
-    }
     return (
       <div className="container-fluid container-pf-nav-pf-vertical">
         <h2>Services</h2>
```

Comment:
see the {alertsDiv} below - as per my comment above, this should at minimum be renamed or just hardcode a \<div/> here - or I think maybe even just remove this entirely.